lib/active_stash/matchers.rb

```diff
@@ -1,13 +1,15 @@
 require_relative "./assess"
 
 if defined?(RSpec)
-  RSpec::Matchers.define :encrypt_sensitive_fields do
+  RSpec::Matchers.define :encrypt_sensitive_fields do |assess|
```

Comment:
Adding an optional `Assess` instance to the args here to make testing the matcher easier.